build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+import net.minecraftforge.gradle.user.tweakers.ClientTweaker
+import net.minecraftforge.gradle.user.tweakers.TweakerExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.spongepowered.asm.gradle.plugins.MixinGradlePlugin
+
+buildscript {
+    repositories {
+        maven { setUrl("http://files.minecraftforge.net/maven") }
+        maven { setUrl("http://repo.spongepowered.org/maven") }
+    }
+
+    dependencies {
+        classpath("net.minecraftforge.gradle:ForgeGradle:${Versions.forge_gradle}")
+        classpath("org.spongepowered:mixingradle:${Versions.mixin_gradle}")
+    }
+}
+
+group = "cc.hyperium"
+version = "1.0-SNAPSHOT"
+
+plugins {
+    kotlin("jvm") version Versions.kotlin
```

Comment:
Can this be merged with the `plugins` block in the other build.gradle.kts file?

Reply:
No, we cannot, as they are two different modules, and `buildSrc` uses a different plugin.